app/helpers/help_center_helper.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module HelpCenterHelper
+  def article_content_classes = "scoped-tailwind-preflight prose dark:prose-invert [&_table,&_tr]:grid [&_table]:gap-4 [&_tbody]:contents [&_tr]:border [&_tr]:rounded [&_td,&_th]:not-last:border-b [&_table]:lg:table! [&_table]:lg:border-collapse [&_table]:lg:border [&_tbody]:lg:table-row-group [&_tr]:lg:table-row [&_td,&_th]:lg:border-y [&_td,&_th]:p-4"
```

Comment:
I don't think it's worth migrating all the help center articles to React here, and it would be overcomplicating things to try and add a table component for React on Rails IMO.

Also note that our current mobile styles are equally broken in the help center, so this is something we should clean up later anyway:



These styles mean that for now plain `<table>` elements in the help articles look equivalently bad on mobile:



And fine on desktop: